library/src/test/java/com/datatorrent/lib/io/fs/FileSplitterBaseTest.java

```diff
@@ -97,7 +97,13 @@ protected void starting(org.junit.runner.Description description)
     @Override
     protected void finished(Description description)
     {
-      TestUtils.deleteTargetTestClassFolder(description);
+      try {
+        //got RuntimeException: java.io.IOException: Unable to delete directory target/com.datatorrent.lib.io.fs.FileSplitterBaseTest/testSplitterInApp
+        //in build server and sometime in local machine
+        TestUtils.deleteTargetTestClassFolder(description);
```

Comment:
deleteQuietly is method of FileUtils, which means we need to duplicate the logic of "new File("target/" + description.getClassName())"